### PR TITLE
[docs] Align Development Stages in HW Dashboard

### PR DESCRIPTION
**Pull request by lenary**
This makes it much easier to read down a single column and see how far
along development and maturity of a given IP actually is.

Columns contain the same links/text/hovertext as before, or `-` if the
HW has not reached any development stage of that kind already.

---

Preview of this change, visually:

**Before**:
Screen Shot 2020-11-17 at 5 32 01 pm

**After**:
Screen Shot 2020-11-17 at 6 49 05 pm

### CODE REVIEW

**Review by gkelly — APPROVED** [review]
Oh that's so much easier to read, thanks for this @lenary.

**Comment by lenary** [comment]
Most recent update changes the table to reduce the amount of vertical lines, while keeping stages aligned

Screen Shot 2020-11-17 at 6 49 05 pm

File: util/dashboard/gen_dashboard_entry.py [hunk 1]
```diff
@@ -229,12 +216,13 @@ def print_version1_format(obj, outfile):
...
+    genout(outfile, "        <td class=\"dv-plan\">" +
                     get_linked_dv_plan(obj) + "</td>\n")
-    genout(outfile, "        <td class=\"hw-stage\">" +
+    genout(outfile, "        <td class=\"version\">" +
                     get_linked_version(obj) + "</td>\n")
-    genout(outfile, "        <td class=\"hw-stage\"><span class='hw-stage'>" +
-                    get_development_stage(obj, obj) + "</span></td>\n")
+
+    for stage_html in get_development_stage(obj, obj):
+        genout(outfile, "        <td class=\"hw-stage\"><span class='hw-stage'>" + stage_html + "</span></td>\n")
```
Comment (sriyerg) [inline util/dashboard/gen_dashboard_entry.py:225]: nit: this probably exceed 80 chars / line.

**Review by sriyerg — APPROVED** [review]
Looks great!

**Review by msfschaffner — APPROVED** [review]
Looks nice!

File: util/dashboard/gen_dashboard_entry.py (hunk 1 above)
Comment (lenary) [reply util/dashboard/gen_dashboard_entry.py]: I did a minor code reformat, to appease `flake8`.

**Pull request merged**